test/client/computeLedgerHash/index.ts

```diff
@@ -29,7 +30,7 @@ export default <TestSuite>{
     ledger.parentCloseTime = ledger.closeTime
     let hash
     try {
-      hash = client.computeLedgerHash(ledger, {computeTreeHashes: true})
+      hash = computeLedgerHeaderHash(ledger, {computeTreeHashes: true})
```

Comment:
See #1550 for how to make these tests not use the client.

Reply:
Done